### ISSUE TITLE
Make `kmsBaseUrl` used in `kms.js` `createKeystore` a configuration option

### ISSUE THREAD

**Issue by dlongley** [digitalbazaar/bedrock-tokenization#6]
bedrock-tokenization should have to be configured to use a particular WebKMS system via a config var for `kmsBaseUrl`. Right now, `kmsBaseUrl` is based on an assumption that the WebKMS used will be accessible via the same domain as where bedrock-tokenization is installed -- which also presumes that bedrock-tokenization uses a server, which may not be true (but likely is). Notably, the WebKMS system used should be architecturally split from whatever system is running bedrock-tokenization even if the domains are shared externally.

In short, we should make `kmsBaseUrl` a configuration option and require it to be set.

**Comment by dlongley** [comment]
This has been done in bedrock-tokenizer, closing.
